addons/main/XEH_postInit.sqf

```diff
@@ -11,7 +11,7 @@ GVAR(CalloutCacheNamespace) = call CBA_fnc_createNamespace;
         if !(".p3d" in _string) then {
             _string = _string + ".p3d";
         };
-        GVAR(blockSuppressionModelCache) setVariable [toLower _string, true];
+        GVAR(blockSuppressionModelCache) setVariable [toLowerANSI _string, true];
```

Comment:
Here we probably should not do that, in case there is someone using none standard ANSI characters for the file path.

Reply:
As far as I remember, setVariable only supports ASCII characters.

Reply:
```sqf
private _yeet = false call CBA_fnc_createNamespace; 
private _string = "\правда\model.p3d"; 
if ((_string select [0,1]) isEqualTo "\") then { 
 _string = [_string, 1] call CBA_fnc_substr; 
}; 
if !(".p3d" in _string) then { 
 _string = _string + ".p3d"; 
}; 
_yeet setVariable [toLowerANSI _string, true]; 
allVariables _yeet
```

returns `["������������\model.p3d"]`

Reply:
that means obfuscated pbos will not work well here.